python/pyfunc-server/pyfuncserver/protocol/rest/server_sanic.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+
+from merlin.pyfunc import PyFuncOutput
+from orjson import dumps, loads
+from prometheus_client import CollectorRegistry
+from prometheus_client.exposition import choose_encoder
+from sanic import Sanic
+from sanic import json as sanic_json
+from sanic import raw, text
+from sanic.request import Request
+
+from pyfuncserver.config import Config
+from pyfuncserver.model.model import PyFuncModel
+
+
+class HTTPServer:
+    def __init__(
+        self, config: Config, metrics_registry: CollectorRegistry, model: PyFuncModel
+    ):
+
+        self.config = config
+        self.workers = config.workers
+        self.model_manifest = config.model_manifest
+        self.http_port = config.http_port
+        self.metrics_registry = metrics_registry
+        self.registered_models: dict = {}
+        self.register_model(model)
+
+        Sanic.start_method = "fork"
+        self.app = Sanic("pyfunc", dumps=dumps, loads=loads)
+        self.app.update_config({"ACCESS_LOG": False})
+        self.setup_routes()
+
+    def setup_routes(self):
+        @self.app.post(r"/v1/models/<model:([a-zA-Z0-9_-]+):predict>")
+        async def predict_handler(request: Request, model):
+            model = self.get_model(model)
+
+            output = model.predict(request.json, headers=request.headers)
```

Comment:
the current `validate` function which uses orjson, is called in request.json as well. The `loads` method is set in app above.

![image](https://github.com/caraml-dev/merlin/assets/30390872/e8c93326-273b-4442-a212-51b5f6a7da57)